src/poprox_recommender/evaluation/evaluate.py

```diff
@@ -0,0 +1,148 @@
+import csv
+import gzip
+import json
+import logging
+from uuid import UUID
+
+import numpy as np
+import pandas as pd
+from docopt import docopt
+from lenskit.metrics import topn
+from tqdm import tqdm
+
+from poprox_concepts.api.recommendations import RecommendationRequest
+from poprox_concepts.domain import Article, ArticleSet
+from poprox_recommender.data.mind import MindData
+from poprox_recommender.evaluation.metrics import rank_biased_overlap
+from poprox_recommender.logging_config import setup_logging
+from poprox_recommender.paths import project_root
+
+logger = logging.getLogger("poprox_recommender.evaluate_offline")
```

Comment:
Let's make the logger names match the package name (`poprox_recommender.evaluation.evaluate`).